HTMLCSSJS/HTMLCSSJShomework1.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head><link rel="stylesheet" href="normalize.css">
```

Comment:
There is no file `normalize.css` in your source code. So it causes the browser occurs an error